### PR TITLE
Emit PluginResponse objects instead of plain janus messages in plugin.emit('message')

### PR DESCRIPTION
**Pull request by vogdb**
Currently when we return from `plugin.sendWithTransaction` an instance of PluginResponse, which has very nice methods to access fields from plain janus messages. On the other hand we emit plain janus messages on `message` events. Inconsistency.

It's an opened discussion and I welcome here any opinion.

@zazabe ?

### CODE REVIEW

**Comment by vogdb** [comment]
@zazabe please review

**Comment by vogdb** [comment]
@zazabe please hold a bit. I'm gonna add another commit

**Comment by vogdb** [comment]
@zazabe please review

**Comment by vogdb** [comment]
I agree that `JanusMessage.getMessage` is not so good method name :( Shall we name it `getPlainMessage`?

**Comment by zazabe** [comment]
`getPlainMessage` 👍

**Comment by vogdb** [comment]
@zazabe please review

File: test/integration/audiobridge.js [hunk 1]
```diff
@@ -49,15 +49,15 @@ describe('Audiobridge tests', function() {
     var roomId = randomRoomId();
     return audiobridgePlugin.create(roomId)
       .then(function(response) {
-        assert.equal(response.getData('audiobridge'), 'created');
+        assert.equal(response.getPluginData(audiobridgePlugin.getResponseAlias()), 'created');
```
Comment (vogdb) [inline test/integration/audiobridge.js]: Everything is possible, master :feelsgood:

**Comment by vogdb** [comment]
@zazabe please review

**Comment by zazabe** [comment]
lgtm

**Pull request merged**